qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -361,6 +361,8 @@ def _init_profiles():
 
     default_profile = QWebEngineProfile.defaultProfile()
     init_user_agent()
+    manager = notification.DBusNotificationManager()
+    manager.set_as_presenter_for(default_profile)
 
```

Comment:
This would probably fit better in `webenginetab.init` (even though it's not per-tab, but I haven't found a better place for such QtWebEngine-specific initialization yet), where various similar stuff is done.